lib/blockscore/connection.rb

```diff
@@ -1,5 +1,11 @@
 module BlockScore
   module Connection
+    MAJOR_VERSION = BlockScore::VERSION.to_i
+    REPO          = 'https://github.com/BlockScore/blockscore-ruby'
+    ACCEPT_HEADER = "application/vnd.blockscore+json;version=#{MAJOR_VERSION}"
+    USER_AGENT    = "blockscore-ruby/#{BlockScore::VERSION} (#{REPO})"
+    CONTENT_TYPE  = 'application/json'
```

Comment:
At present there is no pref in your style guide, and this gemspec does not mention ice nine, adamantium, etc. DO you want them added where appropriate?